devicetypes/stelpro/stelpro-ki-thermostat.src/stelpro-ki-thermostat.groovy

```diff
@@ -161,6 +167,7 @@ def updated() {
 
 	unschedule(scheduledUpdateWeather)
 	if (settings.zipcode) {
+		state.invalidZip = false // Reset and validate the zip-code later
```

Comment:
Unless I'm not understanding this change correctly, I think setting `state.invalidZip = false` just means it will always use lat/lon and not zip due to the check on line 220

Reply:
The check on line 220 verifies if the zip code is not invalid (double negative), so it's a check for validity.
If the zipcode is valid, getTwcConditions() is invoked

Reply:
🤦‍♂️ it's too early, thanks :)